css-layout-classes/style.css

```diff
@@ -0,0 +1,75 @@
+/* font default for entire webpage */
+body {
+  font-family: sans-serif;
+  background: #ddd;
+}
+h1 {
+  text-align: center;
+  font-weight: bold;
+}
+
+h2 {
+  text-align: center;
+  font-size: 20px;
+  font-weight: bolder;
+}
+
+/* flexing the whole flexbox container div */
+#flexbox-container {
+  display: flex;
+  justify-content: center;
+}
+
+#flexbox-container > .flex-para {
+  min-width: 250px;
+  max-width: 450px;
+}
+
+.flex-para-one {
+  min-width: 250px;
+  max-width: 180px;
+}
+
```

Comment:
You shouldn't be setting a `min-width` or a `max-width` on these. You should either be using `flex-basis` or `width` to control the sizing of your columns